.circleci/config.yml

```diff
@@ -1,7 +1,8 @@
 version: 2.1
 jobs:
   test-backend:
-    machine: true
+    machine:
+      docker_layer_caching: false
```

Comment:
@RafterGit I noticed that the two other instances of the `docker_layer_caching` key were nested under the `setup_remote_docker` step; wanted to double check and ask if `docker_layer_caching` can be set at both the `setup_remote_docker` step and the `machine` step? 

Also, I believe `docker_layer_caching` is disabled by default, so wondering if this line is needed?